### PR TITLE
Advertise `docker compose` for non linux users

### PR DESCRIPTION
**Pull request by ulyssessouza**
This adds messages on:
- Root command (only `docker-compose`)
- Command not found
- `help` command

Message:
Docker Compose is now in the Docker CLI, try `docker compose`

Resolves https://github.com/docker/compose-cli/issues/1149

### CODE REVIEW

**Review by ndeloof — APPROVED** [review]

**Review by aiordache — APPROVED** [review]

**Comment by mkvitnitsky** [comment]
@ulyssessouza , how about this wording - 
Try `docker compose`, the new Docker Compose in the Docker CLI
